src/examples/x9.rb

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env ruby
```

Comment:
A script called `x9`, without comments and with variables like `sm, dg, r_all, dd, sl, r1, dp`.... To be honest, I would expect the content of `src/examples' to be more didactic :wink: